src/edfi_amt_data_lake/parquet/amt/asmt/assessment_student/main.py

```diff
@@ -0,0 +1,140 @@
+        leftOn=['id'],
+        rigthOn=['_id'],
+        suffixLeft=None,
+        suffixRight=None
+    )
+    
+    performance_levels = jsonNormalize(
+        student_assesssment_content,
+        recordPath=['performanceLevels'],
+        meta=['id',
+            ['performanceLevelDescriptor']],
+        metaPrefix='_',
+        recordPrefix=None,
+        errors='ignore'
+    )
+
+    result_frame = pdMerge(
+        left=result_frame, 
+        right=performance_levels,
+        how='left',
+        leftOn=None,
+        rigthOn=None,
+        suffixLeft=None,
+        suffixRight=None
+    )
+
+    result_frame = result_frame.fillna('')
+
+    result_frame = subset(result_frame, 
+     [
+         'id',
+         'studentAssessmentIdentifier',
+         'administrationDate',
+         'assessmentReference.assessmentIdentifier',
+         'assessmentReference.namespace',
+         'studentReference.studentUniqueId',
+         'whenAssessedGradeLevelDescriptor',
+         'schoolYearTypeReference.schoolYear',
+         'assessmentReportingMethodDescriptor',
+         'result_frameDatatypeTypeDescriptor'
+    ])
+
+    result_frame['AssessedGradeLevel'] = (
+        result_frame['whenAssessedGradeLevelDescriptor'] + '-'
+        + result_frame['studentReference.studentUniqueId'] + '-'
+        + result_frame['studentReference.studentUniqueId']
+    )
+
+    if not result_frame['AssessedGradeLevel'].empty:
+        if len(result_frame['AssessedGradeLevel'].str.split('#')) > 0:
+            result_frame['AssessedGradeLevel'] = result_frame['AssessedGradeLevel'].str.split("#").str.get(1)
+
+    result_frame['StudentObjectiveAssessmentKey'] = (
+        result_frame['studentReference.studentUniqueId']
+    )
+
+    result_frame['StudentUSI'] = (
+        result_frame['studentReference.studentUniqueId']
+    )
+
+    result_frame['StudentSchoolKey'] = (
+        result_frame['studentReference.studentUniqueId']
+    )
+
+    result_frame['result_frameDataType'] = (
+        result_frame['result_frameDatatypeTypeDescriptor']
+    )
+
+    result_frame['Namespace'] = (
+        result_frame['assessmentReference.namespace']
+    )
+
+    result_frame['StudentAssessmentFactKey'] = (
+        result_frame['assessmentReference.assessmentIdentifier']
+    )
+
+    result_frame['StudentAssessmentKey'] = (
+        result_frame['assessmentReference.assessmentIdentifier']
+    )
+
+    result_frame['ObjectiveAssessmentKey'] = (
+        result_frame['assessmentReference.assessmentIdentifier']
+    )
+
+    result_frame['AssessmentKey'] = (
+        result_frame['assessmentReference.assessmentIdentifier']
+    )
+
+    result_frame['AssessmentIdentifier'] = (
+        result_frame['assessmentReference.assessmentIdentifier']
+    )
+
+    result_frame['AssessmentIdentifier'] = (
+        result_frame['administrationDate']
+    )
+ 
+    if not os.path.exists(f"{config('PARQUET_FILES_LOCATION')}"):
+        os.makedirs(f"{config('PARQUET_FILES_LOCATION')}")
```

Comment:
Yes, I think we need this code, but I am thinking we can add it to the pandas wrapper, save parquet function?

Reply:
Actually we don't need this code. It's covered on [BIA-1215](https://tracker.ed-fi.org/browse/BIA-1215). Please just remove.